app/routes/authorization/callback.js

```diff
@@ -0,0 +1,16 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class AuthorizationCallbackRoute extends Route {
+  @service session;
+
+  queryParams = ['code'];
+  beforeModel() {
+    // redirect to index if already authenticated
+    this.session.prohibitAuthentication('index');
+  }
+
+  async model() {
+    this.session.authenticate('authenticator:acm-idm', this.code);
```

Comment:
I think this needs to be `params.code`, right? `this.code` should be undefined here?

Reply:
Yes that seems to be right, I fixed it. I'll open a PR to correct the example in the ember-acm-idm README.

Reply:
Sorry for the sneak review, I'm doing a similar setup change and was using this PR as a reference 😄.